doc/command_ref.rst

```diff
@@ -136,13 +136,16 @@ Options
 
     DNF uses a separate cache for each user under which it executes. The cache for the root user is called the system cache. This switch allows a regular user read-only access to the system cache, which usually is more fresh than the user's and thus he does not have to wait for metadata sync.
 
+``--color=<color>``
+    control whether color is used
+
 ``--comment=<comment>``
     Add a comment to the transaction history.
 
 ``-c <config file>, --config=<config file>``
     Configuration file location.
 
-``--cve=<cves>``
+``--cves=<cves>``
```

Comment:
Can you really ditch the --cve option after it's been in the man page and it was actually working (due to the nasty [argument abbreviation misfeature](https://docs.python.org/3.5/library/argparse.html#prefix-matching) of python's argparse module). What I did in my patches was add the --cve and also --bz (same situation as with --cve, which you've missed) to the option parser as valid options. I'd prefer to just use --cves and --bzs and not add the --cve and --bz, but I'm afraid someone, somewhere, is already using them...

Reply:
Hi Lukas, thanks for reviewing this. I wasn't aware of the bug, I made this PR long before, when I was asked to fix inconsistencies between --help and docs. Considering the bug now, I added both options to the docs (bz and bzs, cve and cves). Is the option parser change necessary, when it works as it is?

Reply:
@pkratoch by adding it to the option parser you add it to the --help output to make it consistent with the man page. Also, after this gets merged I'm planning to post a PR to disable the argument abbreviations (sadly only for python >=3.5, but that should be everything except RHEL7 anyway), so that this mess doesn't happen anymore :smile:

Reply:
We should change `--bz` to `--bug`/--issue`, as I'm not sure this feature is bugzilla specific...

Reply:
Ok, I'll add it to the option parser.

As for the argument abbreviations disabling, I am afraid it will cause problems for users that were used to typing shortened commands. Some of it can be eliminated by https://github.com/rpm-software-management/dnf/pull/1292 , but I am not sure if it's sufficient. But we can discuss it in your future PR. Also, if it gets merged, we should remove the workaround for some plugin options that would be ambiguous now (--enable, --disable, and I believe there were others).

Reply:
@Conan-Kudo I tried to test this, but didn't figure out how to get some bug IDs that would work...

However, it looks like you are passing a bare bugzilla ID, which actually has to be bugzilla specific, you if happened to have another issue tracker, the IDs could collide.

Reply:
@pkratoch I was afraid it might be the case, although @m-blaha said it might be fine, so I went for it. Yeah, let's discuss it on that future PR.